python/tests/test_pyvw.py

```diff
@@ -353,6 +353,58 @@ def test_example_features():
     assert ex.pop_namespace()
 
 
+def count_weights_from_readable_model_file(file_name):
```

Comment:
Can we add a test that checks the no-slot-features case too?

Reply:
And maybe also validate the model version in the file if it's not too difficult to get

Reply:
The current case is with no slot features, I can check the case where this does not apply when there are slot features. I'm not sure if checking the version makes sense though as that will change on each release